source/modules/models/address.md.erb

```diff
@@ -3,6 +3,20 @@
 Residential or billing address information that is used in other models ([InspetorAccount](#inspetoraccount), [InspetorEvent](#inspetorevent) and [InspetorCreditCard](#inspetorcreditcard)).
 
 ### Properties
+```json
+
+'{
+    "street_name":   "Rua Fidalga",
+    "street_number": "252",
+    "zip_code":      "05432-010",
+    "city":          "São Paulo",
+    "state":         "SP",
+    "country":       "Brasil",
+    "latitude":      "-46.234501",
```

Comment:
In [this other line](https://github.com/inspetor/docs-backend/pull/6/files#diff-a680d207035910577fdb5b0109d8819bR10) latitude and longitude are numbers not string. Is it supposed to be different here?

Reply:
ah, nope--but this is not currently part of the docs anyway

Reply:
(fixed regardless)